doc/source/user_guide/troubleshoot.rst

```diff
@@ -548,4 +540,59 @@ to one, have the same shape if later retrieved.
       [3.],
       [4.]])
    >>> np.allclose(mapdl.parameters['mapdlarray'], mapdl.parameters['mapdlarray_b'])
-   True
\ No newline at end of file
+   True
+
+
+
+.. _ref_pymapdl_stability:
+
+*****************
+PyMAPDL stability
+*****************
+
+Recommendations
+===============
+
+When connecting to an instance of MAPDL using gRPC (default), there are some cases
+where the MAPDL server might exit unexpectedly. There
+are several ways to improve MADPL performance and stability:
+
+- When possible, pass ``mute=True`` to individual MAPDL commands or
+  set it globally with the :func:`Mapdl.mute
+  <ansys.mapdl.core.mapdl_grpc.MapdlGrpc>` method. This disables streaming
+  back the response from MAPDL for each command and marginally
+  improves performance and stability. Consider having a debug flag in
+  your program or script so that you can turn on and off logging and
+  verbosity as needed.
```

Comment:
```suggestion

Use ``mute`` to improve stability
-----------------------------------

When possible, pass ``mute=True`` to individual MAPDL commands or
set it globally with the :func:`Mapdl.mute
<ansys.mapdl.core.mapdl_grpc.MapdlGrpc>` method. This disables streaming
back the response from MAPDL for each command and marginally
improves performance and stability. Consider having a debug flag in
your program or script so that you can turn on and off logging and
verbosity as needed.
```